tools/plugins/com.liferay.ide.gradle.core/META-INF/MANIFEST.MF

```diff
@@ -13,12 +13,12 @@ Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.debug.core,
  org.eclipse.jface,
  biz.aQute.bndlib;bundle-version="3.3.0",
- org.eclipse.buildship.core;bundle-version="1.0.21",
- com.gradleware.tooling.model;bundle-version="0.16.0",
+ org.eclipse.buildship.core,
+ com.gradleware.tooling.model,
  com.liferay.ide.project.core,
  com.liferay.blade.api;bundle-version="2.0.0",
- com.gradleware.tooling.client;bundle-version="0.16.0",
- com.gradleware.tooling.utils;bundle-version="0.16.0",
+ com.gradleware.tooling.client,
```

Comment:
Also lets add versions here, so we can make sure we are using the right versions.